### PR TITLE
Update TaskFollowPointRoute.md

### PR DESCRIPTION
**Pull request by Wetter42**
Hopefully this follows the recent advice given, nonetheless, roastme! :D

Added flags and updated the name of the intunknown

Before you submit this PR, please make sure:

- You have read the contribution guidelines
- You include an example that validates your change
- Your English is grammatically correct

### CODE REVIEW

**Comment by Wetter42** [comment]
RIP - I guess thats a no... 😢

**Comment by ahcenezdh** [comment]
The last argument should be `mode` and you have to declare the enum by wrap it into a c block ```c // Enum here ```
You could move the description of the native at the top 

> Makes the ped go on the created point route

**Comment by ahcenezdh** [comment]
CI failed because you didn't write followpointroutemode in the same way as in the parameters. Anyway, the correct name of this parameter is `mode`.

**Comment by Wetter42** [comment]
Done - Added changes.  please approve for moneys

File: TASK/TaskFollowPointRoute.md [hunk 1]
```diff
@@ -5,8 +5,15 @@ ns: TASK
 
 ```c
 // 0x595583281858626E 0xB837C816
-void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int unknown);
+void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int mode);
```
Comment (AvarianKnight) [inline TASK/TaskFollowPointRoute.md:8]: `routeMode` would be a better name

File: TASK/TaskFollowPointRoute.md [hunk 2]
```diff
@@ -5,8 +5,15 @@ ns: TASK
...
 ```
+### Flags
```
Comment (AvarianKnight) [inline TASK/TaskFollowPointRoute.md:10]: This should be switched to a C enum

```c
enum eFollowPointRouteMode {
	TICKET_SINGLE = 0,
	TICKET_RETURN = 1,
	TICKET_SEASON = 2,
	TICKET_LOOP = 3
}
```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: TASK/TaskFollowPointRoute.md (hunk 2 above)
Comment (Wetter42) [reply TASK/TaskFollowPointRoute.md:10]: Roger that! I was juuuust making the change.  Would something like this work:
````c // enum FOLLOWPOINTROUTEMODE { 
    TICKET_SINGLE=0,    //Stop at the end of the route 
    TICKET_RETURN=1,      //Go once then return back once following the route backwards 
    TICKET_SEASON=2,      //As above but for forever 
    TICKET_LOOP=3         //Treat the points as a loop, i.e follow it to the end then directly back to the first position, forever. 
};`

Or should I add 'endenum?'  Sorry - I'm mostly unfamiliar with c++ syntax

Comment (Wetter42) [reply TASK/TaskFollowPointRoute.md:10]: Geez that was painful to post...but I got it! Thanks for ur patience!

Comment (AvarianKnight) [reply TASK/TaskFollowPointRoute.md:10]: That is not a C enum.

Comment (Wetter42) [reply TASK/TaskFollowPointRoute.md:10]: Ah, so I should just declare enum exactly as you did without any type of format sanitizers like ` or ```?

E.g.
```enum eFollowPointRouteMode {
	TICKET_SINGLE = 0,
	TICKET_RETURN = 1,
	TICKET_SEASON = 2,
	TICKET_LOOP = 3
}```

I'm tryin' here! LOL seriously, thanks 4 ur patience

Comment (Wetter42) [reply TASK/TaskFollowPointRoute.md:10]: Made the changes.  Let me know what you think

File: TASK/TaskFollowPointRoute.md [hunk 3]
```diff
@@ -5,8 +5,15 @@ ns: TASK
 
 ```c
 // 0x595583281858626E 0xB837C816
-void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int unknown);
+void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int routeMode);
 ```
+### Flags
```
Comment (AvarianKnight) [inline TASK/TaskFollowPointRoute.md:10]: You don't need the `### Flags` here

the enum should be surrounded in fences/code blocks
\```c
// put enum here
\```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: TASK/TaskFollowPointRoute.md (hunk 3 above)
Comment (AvarianKnight) [reply TASK/TaskFollowPointRoute.md:10]: You should also get rid of the previous comment thats in a code block and just leave the description they have

File: TASK/TaskFollowPointRoute.md [hunk 4]
```diff
@@ -5,24 +5,36 @@ ns: TASK
...
-int: Unknown. Can be 0, 1, 2 or 3.
-Example:
-TASK_FLUSH_ROUTE();
-TASK_EXTEND_ROUTE(0f, 0f, 70f);
-TASK_EXTEND_ROUTE(10f, 0f, 70f);
-TASK_EXTEND_ROUTE(10f, 10f, 70f);
-TASK_FOLLOW_POINT_ROUTE(GET_PLAYER_PED(), 1f, 0);
+Makes the ped go on a point route.
+
+### Flags
```
Comment (AvarianKnight) [inline TASK/TaskFollowPointRoute.md:13]:
```suggestion

```

File: TASK/TaskFollowPointRoute.md [hunk 5]
```diff
@@ -5,24 +5,36 @@ ns: TASK
...
+	TICKET_RETURN = 1,
+	TICKET_SEASON = 2,
+	TICKET_LOOP = 3
+}
 ```
 
+### Related natives
+* [`TASK_FLUSH_ROUTE`](#_0x841142A1376E9006)
+* [`TASK_EXTEND_ROUTE`](#_0x1E7889778264843A)
+
```
Comment (AvarianKnight) [inline TASK/TaskFollowPointRoute.md:26]:
```suggestion
This native is often times used with [`TASK_FLUSH_ROUTE`](#_0x841142A1376E9006) and [`TASK_EXTEND_ROUTE`](#_0x1E7889778264843A)

```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Pull request merged**